### ISSUE TITLE
Airboss Groove time start and grading

### ISSUE THREAD

**Issue by madmoney99** [FlightControl-Master/MOOSE#1446]
This will change the way in which Airboss will start the groove time. Instead of in zone OR wings level, the proposed change is in zone AND wings level.  This prevents the groove starting while the plane is still rolling out.  Current code causes LULX calls when rolling into the groove (still in a turn) at the correct distance.  Groove should start when wings are level or crosses the centreline, not when it's inside of an arbitrary zone as it is currently. Groove doesn't start at 0.75nm, rather it starts when wings are level and may be shorter based on the approach speed of the aircraft required to hit between 15-19 seconds.

Feedback on time standards from RW LSO is that between 15-19 seconds is acceptable for groove length time.

**Proposed:**
Groove start 
```
    if rho<=RXX and playerData.step==AIRBOSS.PatternStep.GROOVE_XX and (math.abs(groovedata.Roll)<=4.0 and playerData.unit:IsInZone(self:_GetZoneLineup())) then
```
Unicorn time calculation should be between 15-18.99 seconds:
```
  local TgrooveUnicorn=Tgroove and (Tgroove>=15.0 and Tgroove<=18.99) or false
```

**Existing Code:**
```
  if rho<=RXX and playerData.step==AIRBOSS.PatternStep.GROOVE_XX and (math.abs(groovedata.Roll)<=4.0 or playerData.unit:IsInZone(self:_GetZoneLineup())) then

```
Unicorn time calculation
```
local TgrooveUnicorn=Tgroove and (Tgroove>=16.0 and Tgroove<=18.0) or false
```

**Comment by madmoney99** [comment]
Merged into Develop #1469 Mar 2-21.

**Comment by madmoney99** [comment]
Partially closed into Develop #1469 Mar 2-21.  

- Time in groove start to be reconsidered.  
- Unicorn groove time change accepted.
